packages/build/src/plugins_core/edge_functions/index.js

```diff
@@ -41,7 +41,16 @@ const coreStep = async function ({
   const cacheDirectory =
     !isRunningLocally && featureFlags.edge_functions_cache_cli ? resolve(buildDir, DENO_CLI_CACHE_DIRECTORY) : undefined
 
-  // Edge Bundler expects the dist directory to exist.
+  // Cleaning up the dist directory, in case it has any artifacts from previous
+  // builds.
+  try {
+    await fs.rm(distPath, { recursive: true })
+  } catch {
+    // no-op
+  }
```

Comment:
We could do this instead of the try catch.
```suggestion
  await fs.rm(distPath, { force: true, recursive: true })
```

Reply:
But maybe it is safer to have a try catch.

Reply:
Yeah, I think I feel safer with the `try/catch` 😬